src/server/auth/server/api_server.go

```diff
@@ -2233,9 +2233,10 @@ func (a *apiServer) RevokeAuthTokenInTransaction(txnCtx *txnenv.TransactionConte
 	tokens := a.tokens.ReadWrite(txnCtx.Stm)
 	var tokenInfo auth.TokenInfo
 	if err := tokens.Get(auth.HashToken(req.Token), &tokenInfo); err != nil {
-		if !col.IsErrNotFound(err) {
-			return nil, err
+		if col.IsErrNotFound(err) {
+			return nil, auth.ErrRevokeUnknownToken
 		}
```

Comment:
I almost wonder if the old code was just supposed to return here in the case where the token isn't found (making `DeleteToken` idempotent); that it doesn't might be a bug. The fix in this PR makes sense to me, though.

Reply:
Yeah I could see this originally just silently dropping `col.IsErrNotFound`.